x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/InferencePlugin.java

```diff
@@ -138,7 +139,7 @@ public List<ExecutorBuilder<?>> getExecutorBuilders(Settings settingsToUse) {
             new ScalingExecutorBuilder(
                 UTILITY_THREAD_POOL_NAME,
                 0,
-                1,
+                10,
```

Comment:
This deadlocks with only 1 thread available because it gets used by the queue so bumping it up.